spec/models/term_spec.rb

```diff
@@ -3,9 +3,22 @@
 require "rails_helper"
 
 RSpec.describe Term, type: :model do
+  #Test_Word :
+  it { should have_db_index :word }
+  it { should validate_presence_of(:word) }
+  it { should validate_length_of(:word).is_at_most(255) }
   it { should have_db_column(:word).with_options null: false }
+
+  #Test_Meaning :
+  it { should validate_presence_of(:meaning) }
+  it { should validate_length_of(:meaning).is_at_most(255) }
   it { should have_db_column(:meaning).with_options null: true }
+
+  #Test_Reading :
+  it { should validate_presence_of(:reading) }
+  it { should validate_length_of(:reading).is_at_most(255) }
   it { should have_db_column(:reading).with_options null: true }
-  it { should have_db_index :word }
+  #end.
```

Comment:
The block comment style must strictly follow the rule: 
- Must have `begin` before the `end`
- Must indicate with specific block flag
- Must have the same convention: `begin|end` | `/Start|/End` | `NOTE: Start|NOTE: End`
```ruby
# /Start Make Underscore
klass_names = self.class.name.split("::")
klass_names.each { |string| string.downcase! }
klass_names.join("_")
# /End Make Underscore
```